src/packages/migrate/src/utils/seed.ts

```diff
@@ -1,6 +1,35 @@
 import fs from 'fs'
 import path from 'path'
 import execa from 'execa'
+import resolvePkg from 'resolve-pkg'
+import hasYarn from 'has-yarn'
+import chalk from 'chalk'
+import globalDirectories from 'global-dirs'
+
+export function isPackageInstalledGlobally(
+  packageName: string,
+): 'npm' | 'yarn' | false {
+  try {
+    const usingGlobalYarn = fs.existsSync(
+      path.join(globalDirectories.npm.packages, packageName),
+    )
+    const usingGlobalNpm = fs.existsSync(
+      path.join(globalDirectories.npm.packages, packageName),
```

Comment:
```suggestion
      packageDir
```

Reply:
Thanks, looks like the path is 2 times npm and one should have yarn :palmface: will fix